src/main/java/org/arl/fjage/connectors/WebServer.java

```diff
@@ -443,9 +450,18 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
           filename = URLEncoder.encode(filename, String.valueOf(StandardCharsets.UTF_8));
           Path outputFile = outputDir.resolve(filename);
           try (InputStream inputStream = part.getInputStream();
-               OutputStream outputStream = Files.newOutputStream(outputFile, StandardOpenOption.CREATE, StandardOpenOption.TRUNCATE_EXISTING)) {
+               FileOutputStream outputStream = new FileOutputStream(outputFile.toFile())) {
             IO.copy(inputStream, outputStream);
+            // Force data and metadata to disk
+            outputStream.getChannel().force(true);
+            // Ensure lowest-level disk synchronization
+            outputStream.getFD().sync();
```

Comment:
I believe the `outputStream.getChannel().force(true);` should be enough? From the [docs](https://docs.oracle.com/javase/8/docs/api/java/nio/channels/FileChannel.html#force-boolean-):

> If this channel's file resides on a local storage device then when this method returns it is guaranteed that all changes made to the file since this channel was created, or since this method was last invoked, will have been written to that device. This is useful for ensuring that critical information is not lost in the event of a system crash.

Reply:
> If the file does not reside on a local device then no such guarantee is made.

I am not sure what "local device" actually implies here. Would a disk attached over USB be considered "local"?